etcd/etcd-live-http-to-https-migration.md

```diff
@@ -1,73 +1,73 @@
-# etcd Live HTTP to HTTPS migration
+# Enabling HTTPS in an Existing etcd Cluster
```

Comment:
General comments: as discussed, I think we should have two versions of this, one tracking 0.x and one tracking 2.x.  Also I just noticed that this doc is in coreos/docs rather than coreos/etc/Documentation -- I think it really belongs in the latter place.